build.gradle

```diff
@@ -141,6 +141,8 @@ dependencies {
 		exclude module: 'guava'
 		exclude module: 'gson'
 	}
+
+	compile group: 'com.carrotsearch', name: 'hppc', version: '0.7.1'

```

Comment:
Ask @sirsavary exactly why, I initially didn't even notice he added that (not sure why, maybe I was a bit tired). I initially thought he used FastUtil ones (that's what I get for not going over list of imports and missing a change to build.gradle).

And it doesn't make a huge difference - the jar file is already 15MB in size because of MapDB.

After more research - they seem mostly equivalent so I would go with FastUtil.